lib/beacon/content.ex

```diff
@@ -3927,6 +3928,186 @@ defmodule Beacon.Content do
     repo(site).delete(live_data_assign)
   end
 
+  @doc """
+  Creates a new info handler for creating shared handle_info callbacks.
+
+  ## Example
+
+      iex> create_info_handler(%{site: "my_site", msg: "{:new_msg, arg}", code: "{:noreply, socket}"})
+      {:ok, %InfoHandler{}}
+
+  """
+  @doc type: :info_handlers
+  @spec create_info_handler(map()) :: {:ok, InfoHandler.t()} | {:error, Changeset.t()}
+  def create_info_handler(attrs) do
+    variable_names =
+      attrs
+      |> ExUtils.Map.atomize_keys()
```

Comment:
Understood 👍   I got rid of `ExUtils` (its usage, in mix.exs, and in mix.lock). I also removed `list_variable_names/1`. I refactored `retrieve_msg/2` to get the `msg` value whether it arrives as a map's string key, as a map's atom key, or (only if necessary when updating) as the existing info_handler's `msg` attribute. 

I also added `tap(&maybe_broadcast_updated_content_event(&1, :info_handler))`  to `delete_info_handler/1`, because that was missing.

Reply:
> I also added tap(&maybe_broadcast_updated_content_event(&1, :info_handler))

Well observed 👍🏻